.pre-commit-config.yaml

```diff
@@ -14,13 +14,14 @@ repos:
 
       - id: check-json
       - id: check-yaml
-        exclude: ^(deploy/.*|chart/templates/.*)$
+        exclude: ^(deploy/.*|chart/templates/.*|chart/.snapshots/.*)$
```

Comment:
I do think it makes sense to validate the generated snapshots for valid yaml. 

What was the problem that made you exclude the snapshots?